### PR TITLE
Cs/note updates

### PR DESCRIPTION
**Pull request by Mario-Fun**

### CODE REVIEW

**Comment by AlexFine** [comment]
Two general comments here: 

1. Readme Update. We need to update the readme here away from the React template. 
2. Clearer file structure. To an outside developer the file structure is unclear. As a web2 developer I immediately go to the `index.js`, however, that file does not seem to be the relevant file. Perhaps improved documentation would fix this issue. 

Difficult for me to judge in more detail until these two topics are addressed.

**Comment by AlexFine** [comment]
Some implementation comments: 

- Commit language. We need to improve the clarity of code commits. Current commits are not descriptive or informative. This reflects poorly on us as an organization, and does not induce trust. We need an extreme attention to detail on every part of the codebase. Customers are trusting us with their money. That is hugely important.

File: src/App.js [hunk 1]
```diff
@@ -1,65 +1,92 @@
...
-    
+  const [aTokenAddress, setATokenAddress] = useState("0xC42f40B7E22bcca66B3EE22F3ACb86d24C997CC2") // Avalanche Fuji AAVE Dai
+  const [pkey, setPkey] = useState("") // avax-fuji
+  const [walletType, setWalletType] = useState(false)
+
+  // useEffect(() => {
+  //   if (pkey) {
+
+  //   }
+  // }, [pkey])
```
Comment (mariobaxter) [inline src/App.js:19]: remove comments

File: src/App.js [hunk 2]
```diff
@@ -1,65 +1,92 @@
...
-    
-    const provider = new ethers.providers.Web3Provider(window.ethereum)
-    await provider.send('eth_requestAccounts', []); // <- this promps user to connect metamask
-    const eoa = provider.getSigner();
-    
-    // With privateKey
-    
-    // const privKey = "c5ff68eee74d447b88d5d0dd1d438b37f30a4c0b1e41e9c485c6e2ea282d1489"
-    // // const provider = new ethers.providers.JsonRpcProvider(rpc)
-    // const eoa = new ethers.Wallet(privKey, provider)
```
Comment (mariobaxter) [inline src/App.js:23]: Remove comments

File: src/App.js [hunk 3]
```diff
@@ -1,65 +1,92 @@
...
-    /*
-    Deploy the FunWallet with the withdraw from Aave action.
-    User must store the returned executionHash variable to later execure the Aave withdrawal action
-    */
-    const { receipt: deplomentReceipt, executionHash } = await wallet.deployWallet()
-    console.log("Creation Succesful:\n", deplomentReceipt)
+
+
+    // Create a new FunWallet instance, 
+    const prefundAmt = 0 // eth
```
Comment (mariobaxter) [inline src/App.js:48]: Should be 0.3?

File: src/App.js [hunk 4]
```diff
@@ -1,65 +1,92 @@
...
+    const chain = '43113'
```
Comment (mariobaxter) [inline src/App.js:49]: Should be a global variable

**Review by mariobaxter — CHANGES_REQUESTED** [review]

**Pull request merged**